File: 961 - N-Repeated Element in Size 2N Array.cpp
#include <vector>
#include <unordered_map>

using namespace std;

class Solution {
public:
    int repeatedNTimes(vector<int>& nums) {
        unordered_map<int, int> count;
        for (int n : nums) {
            count[n]++;
            if (count[n] >= 2) {
                return n;
            }
        }
        return 114514;
    }
};

File: 961 - N-Repeated Element in Size 2N Array.java
import java.util.HashMap;
import java.util.Map;

class Solution {
    public int repeatedNTimes(int[] nums) {
        Map<Integer, Integer> count = new HashMap<>();
        for (int n : nums) {
            count.putIfAbsent(n, 0);
            count.put(n, count.get(n) + 1);
            if (count.get(n) >= 2) {
                return n;
            }
        }
        return 123;
    }
}